windows/src/wmi.cc

```diff
@@ -25,9 +27,9 @@ namespace leatherman { namespace windows {
     {
 #ifdef LEATHERMAN_I18N
         // LOCALE: format a pointer as hex for printing an error message.
-        return lth_locale::format("{1} (0x{2,num=hex})", s, hres);
+        return _("{1} (0x{2,num=hex})", s, hres);
 #else
-        return lth_locale::format("%1% (0x%2$#x)", s, hres);
+        return _("%1% (0x%2$#x)", s, hres);
```

Comment:
Yes, I just looked at this again and agree that these lines have to remain like this. Our fix-up code in leatherman::locale::format for non-LEATHERMAN_I18N platforms does not handle the formatting here. I believe this is the one and only place we need this.

I'd say merge this PR unless there are any other concerns.